Wireframe/css/style.css

```diff
@@ -0,0 +1,57 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+    margin: 0;
+    padding: 0 0 80px 0;
+}
+header{
+    text-align: center;
+    padding: 20px;
+    background-color: #f4f1f1;;
+}
+
+.wrapper{
+    padding: 20px;
+    width: 80%;
+    margin: 0 auto;
+}
+
+article:first-child {
+    display: block;
```

Comment:
Articles don't need `display: block`, article elements have this as standard, elements that have `display: block` are known as block-level element. Elements such as all headings, paragraphs and so on all are `display: block` inherently. 

So you could delete this line and it wouldn't affect your code.